src/radical/pilot/utils/session.py

```diff
@@ -91,7 +91,6 @@ def fetch_profiles (sid, dburl=None, src=None, tgt=None, access=None,
     log.debug("Number of pilots in session: %d", num_pilots)
 
     for pilot in pilots:
-
```

Comment:
It is a minor point maybe, by please don't change code outside of the scope you work on.  Please also have a look at our [coding guidelines](https://github.com/radical-cybertools/radical.pilot/wiki/Coding-Guidelines) for max line length, indentation rules etc.

Reply:
Read and trying to understand the python things here. I will try to enforce the PIP, pylint with flake8 thing in my dev env.